packages/api/endpoints/granules.js

```diff
@@ -274,19 +275,26 @@ async function del(req, res) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function get(req, res) {
+  const {
+    granulePgModel = new GranulePgModel(),
+    knex = await getKnexClient(),
+  } = req.testContext || {};
   const { getRecoveryStatus } = req.query;
   const granuleId = req.params.granuleName;
-  let result;
+  let granule;
   try {
-    result = await (new Granule()).get({ granuleId });
+    granule = await granulePgModel.get(knex, { granule_id: granuleId });
```

Comment:
My only thought is that just getting a granule by `granule_id` doesn't guarantee uniqueness since it is the combo of `granule_id` and `collection_cumulus_id` that makes a granule unique in RDS. However, unless the user is going to pass a `collectionId` that we could use to determine `collection_cumulus_id`, I don't know if we have any option. 

So this may have to stay as is, but just a thought

Reply:
That's an excellent point.

Options:

1) Leave it: our index is currently PartitionKey: collectionId, sort key: granuleId.    We're currently just returning doc client's .get, so in cases of dupilcate collections/granule ID, the return would be up to the db sort given the base model just returns `.first`.   That *may* be a change from the doc client though, I don't recall if it's predictible which result will show up first. 

2) Allow a collectionId:  This seems 'correct'?

3) Return all matches for that ID: this seems not correct.

Reply:
In either case we should settle on a course and decide on if this requires a new PR, ticket, ? or not.

Reply:
1. I'm not sure I follow. The index set up you're referring to is for DynamoDb, not PostgreSQL? Are you just saying that leaving this query as is to search only by `granule_id` means it'll just return the first result? I agree if so, but not sure if that is what you are saying
2. How would we allow a `collectionId` to be passed into the endpoint? It's GET endpoint and the granuleId comes from a path parameter (e.g. GET /granules/<granuleId>). So to support a `collectionId` parameter, we'd need a new path entirely, which is a new endpoint really. As much as that seems warranted, it also seems out of scope here
3. Agreed

Reply:
1) yes, with the caveat that what that result is might be different
2) We'd have to make it not GET.    Agreed it may be out of scope.  If we want #2 we should ticket/prioritize
3) 👍🏻

Reply:
I've added a ticket : https://bugs.earthdata.nasa.gov/browse/CUMULUS-2649 re this concern on the PR, feel free to annotate/update as needed.

Reply:
It's been added to candidates for the next PI.  Closing.